index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const server = express();
+const cors = require('cors');
+const helmet = require('helmet');
+
+const users = require('./users/users');
+const posts = require('./posts/posts');
+
+server.use(express.json(), cors(), helmet() );
+
+server.use('/users', users);
+server.use('/posts', posts);
+
+const errorHandle = (err, req, res, next) => {
+  const errors = {
+    u1: {
```

Comment:
Like this approach. Works great! Nice job implementing it. Tough concept for many here.